Tools/README

```diff
@@ -41,8 +41,7 @@ peg_generator   PEG-based parser generator (pegen) used for new parser.
 
 scripts         A number of useful single-file programs, e.g. tabnanny.py
```

Comment:
No need to fix in this PR, but `tabnanny.py` isn't actually in this directory any more.

Reply:
I updated the doc.